test/correctness/simd_op_check.cpp

```diff
@@ -41,6 +41,26 @@ int my_process_id = 0;
 // width and height of test images
 const int W = 256*3, H = 100;
 
+bool can_run_code() {
+
+  // If we can (target matches host), run the error checking Func.
```

Comment:
Other than the number of spaces for indentation, are there other things in the code that violate the Halide coding convention?